torch/ao/quantization/experimental/observer.py

```diff
@@ -0,0 +1,47 @@
+"""
+This module implements nonuniform observers used to collect statistics about
+the values observed during calibration (PTQ) or training (QAT).
+"""
+
+import torch
+from torch.ao.quantization.observer import ObserverBase
+from typing import Tuple
+
+class NonUniformQuantizationObserverBase(ObserverBase):
```

Comment:
@vkuzo  good point, but since a goal of this project is to demonstrate to non-uniform quant researchers how they can test their method in our codebase, maybe having the middle of the road class makes their job easier?